example/pubspec.yaml

```diff
@@ -17,6 +17,7 @@ dev_dependencies:
     path: ../
   flutter_test:
     sdk: flutter
+  leak_tracker_flutter_testing: any
```

Comment:
You can see the documentation at https://github.com/dart-lang/leak_tracker/blob/main/doc%2Fleak_tracking%2FDETECT.md


I couldn't apply it to the root project. There are other memory leaks in the tests that need to be fixed first